components/apimgt/org.wso2.carbon.apimgt.core/src/test/java/org/wso2/carbon/apimgt/core/dao/impl/ApiDAOImplIT.java

```diff
@@ -596,10 +596,10 @@ public void testGetAPIsByStatus() throws Exception {
             apiDAO.addAPI(api);
         }
 
-        List<API> blockedAPIsSummary = new ArrayList<>();
-        for (int i = 0; i < numberOfBlocked; ++i) {
-            API api = SampleTestObjectCreator.createUniqueAPI().lifeCycleStatus(blockedStatus).build();
-            blockedAPIsSummary.add(SampleTestObjectCreator.getSummaryFromAPI(api));
+        List<API> maintenanceAPIsSummary = new ArrayList<>();
+        for (int i = 0; i < numberOfInMaintenance; ++i) {
```

Comment:
@uvindra : Noted. I went through the code and noticed in the SampleTestObjectCreator "createUniqueAPI", it creates the API. and here the method is used and random API are created using it by changing some of the values like API ID, context.
btw in this test class the numberOfInMaintenance is only set to 1. So basically we are using the loop to run once?